python/semantic_kernel/ai/open_ai/services/open_ai_chat_completion.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+from logging import Logger
+from typing import Any, Optional, List, Tuple
+
+from semantic_kernel.ai.ai_exception import AIException
+from semantic_kernel.ai.chat_request_settings import ChatRequestSettings
+from semantic_kernel.ai.chat_completion_client_base import ChatCompletionClientBase
+from semantic_kernel.diagnostics.verify import Verify
+from semantic_kernel.utils.null_logger import NullLogger
+
+
+class OpenAIChatCompletion(ChatCompletionClientBase):
+    _model_id: str
+    _api_key: str
+    _org_id: Optional[str] = None
+    _log: Logger
+
+    def __init__(
+        self,
+        model_id: str,
+        api_key: str,
+        org_id: Optional[str] = None,
+        log: Optional[Logger] = None,
+    ) -> None:
+        """
+        Initializes a new instance of the OpenAIChatCompletion class.
+
+        Arguments:
+            model_id {str} -- OpenAI model name, see
+                https://platform.openai.com/docs/models
+            api_key {str} -- OpenAI API key, see
+                https://platform.openai.com/account/api-keys
+            org_id {Optional[str]} -- OpenAI organization ID.
+                This is usually optional unless your
+                account belongs to multiple organizations.
+        """
+        self._model_id = model_id
+        self._api_key = api_key
+        self._org_id = org_id
+        self._log = log if log is not None else NullLogger()
+        self._messages = []
+
+    def _setup_open_ai(self) -> Any:
+        import openai
```

Comment:
Is there a reason why this import is not at the top of the file?

Reply:
This might be fine at the top of the file, it felt more natural given we are explicitly changing properties at the module-level. (And, it other applications, I've had this bite me before where I import `openai` and then use it w/ Azure in some context and "regular" in another context but forget to reset these module level vars...)

Reply:
I would leave it local for now. Once we refactor out the backends, the problem will go away